config/vitest.config.ts

```diff
@@ -0,0 +1,7 @@
+import { defineConfig } from "vitest/config";
+
+export default defineConfig({
+  test: {
+    include: ["**/*.spec.js"],
```

Comment:
**Question:** Is it possible to run vitest directly on the TS files? (I've heard yes) Not sure what is best (OTOH, running ts-node and vitest on `.ts(x)` would reduce build time, keeping it only for the publication flow).

Reply:
Yes, that's correct. I initially couldn't get it to work with the `.tsx` files, that's why I decided to run the emitted JS, but for some reason now it just works on both `.ts` and `.tsx`. 😅

We could remove the tests the build altogether as I don't think we need them anymore and we would be able to have a single `.tsconfig` in every package again. Or do you think we still need the emitted JS from the test files?

Reply:
Yes, the idea is to remove the build step. It is needed for `src` since that's what we distribute, but we don't care about it for `test`, so we can simplify code structure a bit.
I think knip will also get less confused to not have the `.js` together with the `.ts(x)` 😅 

If we fully switch to ts-node or the like, we can also totally remove the build step in the normal flow. We'd still need it in the publish flow (and so, for the sake of sanity, keep it in the `integrate` workflow), but local runs could be faster if we don't have to rebuild.

Reply:
(might be in another PR, though, no need to make this one explode)